broker/common/types.go

```diff
@@ -29,10 +29,14 @@ type SchemaManager interface {
 	GetTableSchemaReader(namespace string) (memCom.TableSchemaReader, error)
 }
 
+// QueryContext defines query context
+type QueryContext interface {
```

Comment:
empty interface?

Reply:
nah it's something I forgot to clean up.. thanks for the catch.